internal/k8s/client.go

```diff
@@ -583,9 +585,16 @@ func (w *ClientWrapper) GetService(namespace, name string) (*corev1.Service, boo
 
 // GetServices retrieves the services from the specified namespace.
 func (w *ClientWrapper) GetServices(namespace string) ([]*corev1.Service, error) {
+	return w.GetServicesWithSelectors(namespace, "", "")
+}
+
+// GetServicesWithSelectors retrieves the services from the specified namespace and given selectors.
+func (w *ClientWrapper) GetServicesWithSelectors(namespace string, labelSelectors, fieldSelector string) ([]*corev1.Service, error) {
```

Comment:
Does this need to be added to the interface/mock?

Reply:
I'm  pretty sure I'll remove this at some point.

Reply:
Currently no tests relies on this call.  I would add this only if needed. WDYT ?

Reply:
I ended up removing it.